app/routes/events/components/EventDetail/index.js

```diff
@@ -230,6 +237,32 @@ export default class EventDetail extends Component<Props, State> {
             ),
           }
         : null,
+      activationTimeMoment.isBefore(currentMoment)
+        ? {
+            key: 'Frist for av/påmelding',
+            keyNode: (
+              <Tooltip
+                content={
+                  <span>
+                    Etter påmeldingen stenger er det hverken mulig å melde seg
+                    på eller av arrangementet.
+                  </span>
+                }
+              >
+                {currentMoment.isBefore(registrationCloseTimeMoment)
+                  ? 'Påmelding stenger'
+                  : 'Påmelding stengte'}{' '}
+                <Icon name="help-circle-outline" size={16} />
```

Comment:
Could you flex this [like this](https://github.com/webkom/lego-webapp/pull/2959/files#diff-74791df4b2365e0328f80b092e14dd504f8584ca1d6dcfc829e336d6542071c3R222-R225), so it'll still look good  once that is merged 😅

Reply:
Updated now(: although gap={1} was a bit close for my taste, so I upped it to gap={4} - thoughts about doing the same to your PR?

Reply:
haha, sure, I'll do it right away!